marketpulse/main/static/js/fxosprice_new.js

```diff
@@ -57,6 +57,18 @@ $(document).ready(function() {
         }
     });
 
+    $('.is-fxos input').change(function() {
+      if (this.checked) {
+        $('.other-device').hide();
+        $('.fxos-device').show();
+        $('.other-device input').val('');
+      } else {
+        $('.other-device').show();
+        $('.fxos-device').hide();
+        $('.fxos-device select').val('');
```

Comment:
If you add a default value to the model, probably you don't need this - haven't tested yet though